src/CoreEx/Http/TypedHttpClientBaseT.cs

```diff
@@ -266,6 +284,10 @@ protected override async Task<HttpResponseMessage> SendAsync(HttpRequestMessage
 
                 throw;
             }
+            finally
+            {
+                cts?.Dispose();
```

Comment:
@chullybun `Reset()` is never called in `SendAsync()` - is that by design?

Reply:
It should call Reset() as per our discussion.